kenware/Assessment
# #162187176 create question endpoints

- add all question endpoints to perform crud operation in question
- write test on the question endpoints
- include children in get assessment endpoints
- write query parser for my application

#### What does this PR do?
- create question endpoints

#### Description
- create  POST endpoint
- create GET endpoints
- create PATCH endpoints
- create DELETE endpoint
- add test
#### How should this be manually tested?
- Create a local copy `git ft-create-question-endpoints-162711236`
- Checkout to the branch `git checkout ft-create-question-endpoints-162711236`
- Start the application with the command: `python manage.py runserver`
- On postman make a GET or POST request to `{{base-url}}/api/v1/questions/<question_id>/`
- On postman make a PATCH or DELETE request to `{{base-url}}/api/v1/questions/<question_id>/`
- An authenticated user should be able to carry out all crude operation on user endpoints
- To run tests use `python manage.py`
#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#162187176](https://www.pivotaltracker.com/story/show/162187176)
